examples/plot_driver_laptimes.py

```diff
@@ -0,0 +1,54 @@
+"""Driver Laptimes Scatterplot
+==============================
+
+Plot a driver's lap times in a race.
+"""
+
+import fastf1 as f

```

Comment:
Please use ``import fastf1`` and ``import fastf1.plotting`` without ``as ...``. I don't think that the single letter variables aid readability here. It's also how the other examples are written, so it ensures consistency.